yazi-cli/src/args.rs

```diff
@@ -22,6 +22,10 @@ pub(super) enum Command {
 	PubStatic(CommandPubStatic),
 	/// Manage packages.
 	Pack(CommandPack),
+	/// Subscribe to messages from remote instance(s).
+	Sub(CommandSub),
+	/// Subscribe to messages from all remote instance(s).
+	SubStatic(CommandSubStatic),
```

Comment:
Why do we need to categorize them into "sub" and "sub static"?

Reply:
I'm not sure it's required 🤔 the idea was to be able to filter only the messages from a specific client, mirroring the `pub` and `pub-static` options.

Reply:
This does not apply to static messages, because static messages may come from the client you want to filter, but do not have a client ID (instead they have a severity), so all static messages from the specific client you want to receive will be lost. 

If you want to receive and filter all messages from a specific client (not just non-static messages) then we will need to  change the protocol itself, which was previously covered by ‘yazi --local-events’ and therefore not considered.

Reply:
The support for listening to a specific peer has been dropped now.

Reply:
Should be renamed as `Sub` if you want to listen for all messages rather than just static messages